test/run-tests

```diff
@@ -33,6 +34,35 @@ def sighandler_exit(signo, frame):
     sys.exit(0)
 
 
+def handle_httperror(error):
+    """
+    Try to handle HTTPError.
+    Tries to handle transient errors by waiting.
+
+    Return True if error is handled.
```

Comment:
Maybe change the docstring to something like `Sleep when a transient error occured, return True if slept and False otherwise`. There does not seem to be more handling other than sleeping therefore this could be explicitly mentioned.